src/main/resources/hudson/tasks/junit/JUnitResultArchiver/help-checksName.html

```diff
@@ -1,4 +1,4 @@
 <div>
     If provided, and publishing checks enabled, the plugin will use this name when publishing results to corresponding
-    SCM hosting platforms. If not, a default of "Tests" will be used.
+    SCM hosting platforms. If not, a default including the current stage / branch names will be used.
```

Comment:
This build step would not normally be used in Pipeline jobs. (It is still _possible_, via some rather gritty Groovy syntax, because @abayer left this as a `SimpleBuildStep` after introducing `JUnitResultsStep`, but very unlikely.) So this help would be displayed to users of traditional job types, for which “stage” and “branch” are meaningless.